app/src/main/java/buggy/plandroid/com/plandroidbugs/UserMapper.java

```diff
@@ -16,6 +16,7 @@
             entity.lastName = userWire.lastName;
             entity.language = userWire.language;
             entity.userId = userWire.userId;
+            entity.roleUid = userWire.role != null ? userWire.role.uid : UserAdapter.ADMIN_ROLE;
```

Comment:
This logic from here is not appropriate. Let's discuss with the product what they want to show in case we get no value from the server